front/src/Phaser/Game/GameMap.ts

```diff
@@ -189,6 +189,10 @@ export class GameMap {
         return this.phaserLayers.find((layer) => layer.layer.name === layerName);
     }
 
+    public findPhaserLayers(groupName: string): TilemapLayer[] {
+        return this.phaserLayers.filter((l) => l.layer.name.includes(groupName));
```

Comment:
This could actually include layers not belonging to this group.

For instance, if you have:

- group1
  - foo
- group11
  - bar

`findPhaserLayers('group1')` will return `group11/bar` (because it **starts** with `group1`).

You should search for "group1/" (so the `findPhaserLayers` should add a "/" suffix